### ISSUE TITLE
CPU percentange and Memory usage not showing for ROS2 humble

### ISSUE THREAD

**Issue by Avi241** [ros-visualization/rqt_top#17]
I have compiled the repo for ros2 but it is not showing cpu-percentage and memory it just shows the name of node

**Comment by clalancette** [comment]
Yes, we deprecated `rqt_top` in ROS 2 because it doesn't show useful information.  As of right now, we don't know how to make it show useful information in a platform-agnostic way.  If you are interested in finding ways for it to show that information across Linux, Windows, and macOS, we'd be happy to review patches.

**Comment by Avi241** [comment]
Sure I am actively working on this ..will update if it works.

**Comment by clalancette** [comment]
> Sure I am actively working on this ..will update if it works.

Thank you!

**Comment by luvwinnie** [comment]
Is there any progress on this?

**Comment by Avi241** [comment]
No. I wasn't able to do it. As per my knowledge ROS2 does not exposed the PID of the nodes for security concerns so we are not able to get the Computation useage as well. For workaround I was able to get the PID using top and the using that PID and psutil python library was able to monitor computation usage
